pkgs/development/tools/rgp/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib, stdenv, makeWrapper, fetchurl, autoPatchelfHook,
+  dbus, fontconfig, freetype, glib, libglvnd, libX11, libxcb, libXi, ncurses5,
+  qtbase, zlib
+}:
+with stdenv.lib;
+
+let
+  deps = with pkgs; [
+    dbus
+    fontconfig
+    freetype
+    glib
+    libglvnd
+    libX11
+    libxcb
+    libXi
+    ncurses5
+    qtbase
+    zlib
+  ];
+
+in stdenv.mkDerivation rec {
+  pname = "rgp";
+  version = "1.7.0.29";
+
+  src = fetchurl {
+    url = https://github.com/GPUOpen-Tools/Radeon-GPUProfiler/releases/download/v1.7/RadeonGPUProfiler_1.7.0.29.tgz;
```

Comment:
It would be nice to automatically generate the appropriate URL for `version`. The utility functions in `lib/versions.nix` might help here.